airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -318,7 +357,8 @@ private static Runnable getReplicationRunnable(final AirbyteSource source,
                                                  final MessageTracker messageTracker,
                                                  final Map<String, String> mdc,
                                                  final RecordSchemaValidator recordSchemaValidator,
-                                                 final WorkerMetricReporter metricReporter) {
+                                                 final WorkerMetricReporter metricReporter,
+                                                 final ThreadedTimeHolder timeHolder) {
     return () -> {
       MDC.setContextMap(mdc);
       LOGGER.info("Replication thread started.");
```

Comment:
Would it be more appropriate to track the start time here, when the runnable is actually executed, rather than in the DefaultReplicationWorker?

Reply:
I think that's what's happening above:
`destination.start(destinationConfig, jobRoot)` should be blocking until the docker container starts, and we immediately after start the clock.

Reply:
Aha, got it!